admin/developers_tool_kit.php

```diff
@@ -8,6 +8,8 @@
 require('includes/application_top.php');
 
 $default_context_lines = 0;
+$output = '';
+$outCount = '';
```

Comment:
How about a bit more descriptive variable names, like `$matches_output` and `$matches_header`, respectively.

Reply:
done